src/components/Dashboard/CompleteFiling.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="complete-ia-filing body-2">
+    <h4>Incorporation Complete</h4>
+
+    <p>{{name}} has been successfully incorporated.</p>
+    <p>Return to your Manage Businesses dashboard to access your business and file changes.</p>
+
+    <div class="to-dashboard-container">
+      <v-btn
+        color="primary"
+        @click.native.stop="returnToDashboard()"
+        >
+        <span>Return to Dashboard</span>
+      </v-btn>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { mapState } from 'vuex'
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import { ErrorContact } from '@/components/common'
+
+@Component({
+  components: { ErrorContact },
+  computed: {
+    ...mapState(['entityName', 'entityIncNo'])
+  }
+})
+export default class CompleteFiling extends Vue {
+  readonly entityName!: string
+  readonly entityIncNo!: string
+
+  /** Determine the name for the entity.
+   * Use the entity name if it is available, otherwise build name for numbered company.
+   */
+  private get name (): string {
+    return this.entityName || `${this.entityIncNo.replace('BC', '')} B.C. LTD.`
```

Comment:
What's an example of entityIncNo in this scenario?
Also, you should check that entityIncNo isn't null.

Reply:
I checked a numbered company completed filing, and it comes in with entityIncNo = 'BCxxxx' and the entityName is null.

I will add that check.

Reply:
Is that really the name we display? I thought this referred to number company names?

Reply:
Yes, that is for the numbered company name, otherwise the entity name should be used. When I loaded a numbered company the entityName came back null so I added this piece in so it displays properly for a numbered company.

Reply:
For a numbered company, entityName is not set.

I think the BCxxx number is not the "company number" we're talking about. This is the business number.

The name for a numbered company is the incorp # (assigned on completion in COLIN).

We should discuss this further with Scott.

Reply:
Discussed with Scott, going with "A numbered benefit company"